### ISSUE TITLE
Please make "generate predictions" optional

### ISSUE THREAD

**Issue by graeme-winter** [ccp4/DUI2#23]
Derived from #22 - if you have a non-trivial data set then it seems be impossible to proceed to the next step without predicting based on teh results from a previous one. The prediction is taking a long time and is not necessary (nor are the logs visible) 

Ah, this was in the end of the stdout, is related?

```
tmp_cmd2lst = ['gmi None']
replacing  gmi  with  get_mask_image

 cmd_lst:  [['get_mask_image', 'None']]

  err catch , wrong line, not sending IMG 

sending /*EOF*/
sending /*EOF*/
zlib. err catch (load_img_json_w_str)
show_refl = True
None self.np_full_img
no mask to draw here
zlib. err catch (load_img_json_w_str)
show_refl = True
None self.np_full_img
no mask to draw here

 BrokenPipe err catch  while sending EOF or JSON 
```

**Comment by luisodls** [comment]
same fix as #16

**Comment by luisodls** [comment]
Done
